### DISCUSSION TITLE
Very sad that you are showing Crimea as a part of russia

### DISCUSSION THREAD

**Discussion by Baller-Michael** [sgratzl/chartjs-chart-geo#220] [Show and tell]
![image](https://github.com/sgratzl/chartjs-chart-geo/assets/25987944/de316473-18d2-4741-b290-ccc0eba715b4)

**Comment by sgratzl** [comment]
this plugin for chart.js doesn't provide any topojson files. The given topojson files and their features determine what the plugin is showing and what areas of the world are associated with Russia
